code/datums/gamemode/objectives/target/locate.dm

```diff
@@ -0,0 +1,49 @@
+/datum/objective/target/locate
+	name = "locate objects"
+	var/list/objects_to_locate = list()
+
+/datum/objective/target/locate/IsFulfilled()
+	.=..()
+	if(objects_to_locate.len)
+		return FALSE
+	return TRUE
+
+/datum/objective/target/locate/random/find_target()
+	var/list/potential_objects = list(/obj/item/weapon/bikehorn/rubberducky,
```

Comment:
some limited objects that may be useful here

* cane, muzzle, and/or stethoscope
* defibrillator
* a spray bottle
* the holo switch tool
* the research director's rubber stamp
* a tank transfer valve
* the tank of ethanol
* Poly
* an ansible crystal
* the bar shaker
* the bar sign
* a specific vinyl from the jukebox (e.g.: jazz)
* the original bottle of bleach
* the galoshes
* the vial storage box

Reply:
👍